src/test/java/org/jenkinsci/plugins/docker/workflow/DockerDSLTest.java

```diff
@@ -324,4 +324,20 @@ private static void grep(File dir, String text, String prefix, Set<String> match
             }
         });
     }
+
+    @Test public void run() {
+        story.addStep(new Statement() {
+            @Override public void evaluate() throws Throwable {
+                assumeDocker();
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "prj");
+                p.setDefinition(new CpsFlowDefinition(
+                        "node {\n" +
+                                "     def busybox = docker.image('busybox');\n" +
+                                "     busybox.run('--rm', 'echo \"Hello\"');\n" +
+                                "}", true));
+                WorkflowRun b = story.j.assertBuildStatusSuccess(p.scheduleBuild2(0));
+                story.j.assertLogContains("Hello", b);
```

Comment:
A weak test, since this would pass even if the log were something like

```
+ docker run --rm busybox echo "Hello"
ERROR: incorrect syntax
(but I am going to return exit code 0 anyway)
```